gatorgrade/generate/generate.py

```diff
@@ -14,8 +14,40 @@ def create_targeted_paths_list():
     # Add those targeted file paths into a list and returns it.
 
 
-def write_yaml_of_paths_list():  # expected input: A path list
+def write_yaml_of_paths_list(path_names):  # expected input: A path list
     """Write YAML file to create gatorgrade file and set default messages."""
-    # Create a new YAML file with PyYaml in the specific path.
+
+    setup_dict = {
+        "setup": "# add setup commands here\n",
+    }
     # Write the default set up messages in YAML file.
-    # List the file paths in specific format.
+
+    path_names = create_targeted_paths_list(["test"])
```

Comment:
Hi Jack, nice work creating the write YAML functionality. I just wanted to point out here that when calling the `create_targeted_paths_list` function, you want to provide a second argument like `create_targeted_paths_list(["test"], "../..")` to scan the root directory.

Reply:
I realized I did not need this line of code pointed out by @Yanqiao4396. Thanks for clarification though.